python/lsst/dax/obscore/script/obscore_set_exposure_regions.py

```diff
@@ -89,36 +87,23 @@ def obscore_set_exposure_regions(
 
     butler = Butler(repo, writeable=True)
 
-    # There are no client API for updating obscore table, so we are going to
-    # mess with the table contents directly. For that we need to have an access
-    # to the internals of the Registry and obscore manager.
-
     registry = butler.registry
-    assert isinstance(registry, SqlRegistry), "Registry must be SqlRegistry"
-
-    db = registry._db
-    obscore_mgr = registry._managers.obscore
-    if obscore_mgr is None:
+    if registry.obsCoreTableManager is None:
```

Comment:
```suggestion
    if (obscore_mgr := registry.obsCoreTableManager) is None:
```

or maybe:

```python
obscore_mgr = registry.obsCoreTableManager
if obscore_mgr is None:
    raise ...
```
It seems odd to me that `registry.obsCoreTableManager` appears in two consecutive statements.

Reply:
I tried that, but for some reason mypy does not like that, it complains that `obscore_mgr` could be `None` inside `_count_missing` below. Not sure why, looks like mypy is confused by closures.